web-registry/src/pages/Projects/hooks/useProjectsSellOrders.utils.tsx

```diff
@@ -26,14 +35,22 @@ export const getPurchaseInfo = (
     .reduce((total, quantity) => total + quantity, 0);
 
   const prices = ordersForThisProject
-    .map(order => microToDenom(order.askAmount))
+    .map(order => {
+      const amount = microToDenom(order.askAmount);
+      const denomPrice = order.askDenom === REGEN_DENOM ? regenPrice ?? 0 : 1;
```

Comment:
If the sell order denom is regen then we compute the price otherwise we keep the amount as it is since future denom we'll likely be USD stablecoins.

Reply:
I wonder if we should check if REGEN is an allowed denom before making any queries? This will save us on API usage until REGEN is approved as a mainnet currency.

You can do this with the `Params` query to get `allowedDenoms`